core/src/test/java/nl/nn/adapterframework/pipes/XmlIfTest.java

```diff
@@ -37,7 +37,7 @@ public void nullXPathExpressionTest() throws ConfigurationException, PipeStartEx
 		pipe.configure();
 		pipe.start();
 
-		PipeRunResult pipeRunResult = pipe.doPipe("<test", session);
+		PipeRunResult pipeRunResult = doPipe(pipe, "<test", session);
```

Comment:
het is een beetje onzinnen hier pipe ook mee te geven?

Reply:
behalve als je zelf een pipe wil maken